cli/src/commands/resolve.rs

```diff
@@ -192,7 +192,7 @@ pub(crate) fn print_conflicted_paths(
         write!(formatter, "{formatted_path} ",)?;
         formatter.with_label("conflict_description", |formatter| {
```

Comment:
Is there a reason you haven't done as you did 2 lines down here?

Reply:
If you mean `formatter.labeled()`, it doesn't return new `Formatter`, so you can't `.labeled(x).labeled(y)` for example.

Reply:
Ahh